modules/nf-core/ariba/run/tests/main.nf.test

```diff
@@ -0,0 +1,65 @@
+// nf-core modules test ariba/run
+nextflow_process {
+
+    name "Test Process ARIBA_RUN"
+    script "../main.nf"
+    process "ARIBA_RUN"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ariba"
+    tag "ariba/run"
+
+    test("illumina - fatsq") {
+
+        setup {
+            run("ARIBA_GETREF") {
+                    script "../../../ariba/getref/main.nf"
+                    process {
+                        """
+                        input[0] =  "card"
+                        """
+                    }
+                }
+        }
+        when {
+            process {
+                """
+
+                input[0] = [ [ id:'test', single_end:false ], // meta map
+                        [   file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true),
+                            file(params.test_data['sarscov2']['illumina']['test_2_fastq_gz'], checkIfExists: true) ],
+                        ]
+                input[1] = ARIBA_GETREF.out.db
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(file(process.out.results[0][1][0]).name).match("assembled_genes.fa.gz") },
+                { assert snapshot(file(process.out.results[0][1][1]).name).match("assembled_seqs.fa.gz") },
+                { assert snapshot(file(process.out.results[0][1][2]).name).match("assemblies.fa.gz") },
```

Comment:
why aren't any of these gzipped fasta files generating any consistent md5sums ?

Reply:
@buehlere :point_up:

Reply:
@sateeshperi I think this is because the compression done by ariba keeps the time stamp on the gzipped files:

<img width="1047" alt="image" src="https://github.com/nf-core/modules/assets/31450790/31bab0f2-38e0-4bc0-9b50-2a904d801a80">

I was reading that this can cause problems with the md5sums [here](https://unix.stackexchange.com/questions/31008/why-does-the-gzip-version-of-files-produce-a-different-md5-checksum). This might be why they only checked for the path on the gzipped files in [the original test.](https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/ariba/run/test.yml#L7-L15)?

Is there a way to save the original and re-generated nf-test files for a comparison to prove this? I'm not sure I see each distinct run in the work directory: 

<img width="1009" alt="image" src="https://github.com/nf-core/modules/assets/31450790/090412b8-2000-4f58-bae2-e1af6ea1c784">

@apetkau I maybe remember you hitting a similar issue with time stamps on gz files. Is there a good way to demonstrate this?

Reply:
@buehlere yes, I encountered a similar issue with md5's changing for gzip files. It was resolved for me by using the `-n` option to `gzip`:

> -n
When compressing, do not save the original file name and timestamp by default. (The original name is always saved if the name had to be truncated.) When decompressing, do not restore the original file name if present (remove only the gzip suffix from the compressed file name) and do not restore the original timestamp if present (copy it from the compressed file). This option is the default when decompressing.
https://www.gnu.org/software/gzip/manual/html_node/Invoking-gzip.html

However, I believe `ariba` is gzipping the files internally, so if this was the issue it would have to be fixed within `ariba` itself (or md5 sums ignored for those particular files/other tests performed).